### PR TITLE
Include Git SHA in the /v1/ endpoint

### PR DESCRIPTION
**Pull request by Natim**
Imported from https://github.com/mozilla-services/readinglist/issues/188

### CODE REVIEW

**Comment by ametaireau** [comment]
From [their documentation](https://github.com/klen/dealer#id24), if you actually include their contrib then the git revision is available at the request level (`request.registry.dealer.revision`):

```
config.include('dealer.contrib.pyramid')
```

**Comment by ametaireau** [comment]
Actually, nevermind. What you propose is clearer / cleaner in my opinion.

**Comment by Natim** [comment]
> Actually, nevermind. What you propose is clearer / cleaner in my opinion.

Yes I have seen their helper and I went to the same conclusion.

File: cliquet/views/hello.py [hunk 1]
```diff
@@ -20,6 +21,9 @@ def get_hello(request):
     if eos:
         data['eos'] = eos
 
+    if git and git.revision:
+        data['commit'] = git.revision
```
Comment (leplatrem) [inline cliquet/views/hello.py]: Could you please add a note the endpoints docs ?

Especially, what happens if installing from pip ?

Comment (Natim) [inline cliquet/views/hello.py]: ``TypeError: fatal: Not a git repository (or any parent up to mount point /home)``

**Pull request merged**